sample-apps/spark/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM gradle:6.5.0-jdk11 as builder
+
+# build
```

Comment:
If using docker, recommend using jib instead of docker, can just use this base image which has the agent preloaded.

https://github.com/aws-observability/aws-otel-java-instrumentation/blob/main/otelagent/build.gradle.kts#L83